API_ENDPOINTS.md

```diff
@@ -0,0 +1,994 @@
+    "payloadLength": 19619,
+    "previousBlock": "",
+    "reward": "0",
+    "timestamp": 0,
+    "totalAmount": "10000000000000000",
+    "totalFee": "0",
+    "totalForged": "0",
+    "version": 0,
+    "delegate": {
+      "address": "1330932780504881464L",
+      "approval": 98.15,
+      "missedblocks": 9,
+      "producedblocks": 188,
+      "productivity": 95.43,
+      "publicKey": "68680ca0bcd4676489976837edeac305c34f652e970386013ef26e67589a2516",
+      "rate": 41,
+      "username": "genesis_82",
+      "vote": "9822932090000000"
+    }
+  }
+}
+```
+
+### /api/getBlockStatus
+
+Retrieves node details.
+
+#### Sample request
+
+`/api/getBlockStatus`
+
+#### Sample response
+
+```
+{
+  "success": true,
+  "broadhash": "2768b267ae621a9ed3b3034e2e8a1bed40895c621bbb1bbd613d92b9d24e54b5",
+  "height": 17313,
+  "build": "v13:14:28 17/04/2018\n",
+  "commit": "9a7e1ef9d5ba40e6b150586bf452064bfb0f831e",
+  "epoch": "2016-05-24T17:00:00.000Z",
+  "fees": {
+    "send": "10000000",
+    "vote": "100000000",
+    "secondSignature": "500000000",
+    "delegate": "2500000000",
+    "multisignature": "500000000",
+    "dappRegistration": "2500000000",
+    "dappWithdrawal": "10000000",
+    "dappDeposit": "10000000",
+    "data": "10000000"
+  },
+  "nethash": "198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d",
+  "nonce": "O2wTkjqplHII5wPv",
+  "milestone": 0,
+  "reward": 500000000,
+  "supply": 10007577000000000,
+  "version": "1.0.0",
+  "fee": 10000000
+}
+```
+
+## Common
+
+### /api/getPriceTicker
+
+Retrieves current currency prices.
+
+#### Params
+
+This request has no params
+
+#### Sample request
+`/api/getPriceTicker`
+
+#### Sample response
+```
+{
+  "success": true,
+  "tickers": {
+    "BTC": {
+      "EUR": "7697.62",
+      "RUB": "538494.7906872",
+      "USD": "9390.8"
+    },
+    "LSK": {
+      "BTC": "0.00137299",
+      "EUR": 10.5687552838,
+      "RUB": 739.3479626656188,
+      "USD": 12.893474491999998
+    }
+  }
+}
+```
+
+
+### /api/search
+
+Performs search among the delegates, accounts, public keys, transactions, blocks and height.
+
+#### Params
+
+ - `id` - search query
+
+#### Sample request
+
+`/api/search?id=genesis`
+
+#### Sample response
+
+```
+{
+  "success": true,
+  "result": {
+    "type": "delegate",
+    "delegates": [
+      {
+        "address": "12577917432507761736L",
+        "username": "genesis_7",
+        "similarity": 0.8571428571428571
+      }
+    ]
+  }
+}
+```
+
+## Delegates
+
+### /api/delegates/getActive
+
+Retrieves active delegates list.
+
+#### Sample request
+
+`/api/delegates/getActive`
+
+
+#### Sample response
+
+```
+{
+  "success": true,
+  "delegates": [],
+  "totalCount": 403
+}
+```
+
+### /api/delegates/getStandby
+
+Retrieves standby delegates list.
+
+#### Params
+
+ - `n` - ranking number of the first item (offset)
+
+#### Sample request
+
+`/api/delegates/getStandby?n=0`
+
+
+#### Sample response
+
+```
+{
+  "success": true,
+  "delegates": [],
```

Comment:
Also, I'd suggest to describe the structure of members of this array in a comment